be/db/dev-db/sql/create_tables.sql

```diff
@@ -0,0 +1,35 @@
+-- template from: https://github.com/jdaarevalo/docker_postgres_with_data
```

Comment:
i feel like this is not needed atm as the tables and data should be created by some js migration/orm library. Its fine for now as it might be helpful for initial testing.